installation/crd.yaml

```diff
@@ -77,3 +77,8 @@ spec:
                     dataKey:
                       type: string
                       description: The key to use in Kubernetes secret when setting the value from Azure Keyv Vault object data
+                    chainOrder:
+                      type: string
+                      description: Changes order of certificates
+                      enum:
+                      - ensureserverfirst
```

Comment:
Not possible for you to know, but this crd is not in use. It's the CRD that's part of the Helm chart in https://github.com/sparebankenvest/public-helm-charts that's used. 

I can update that repo, and just leave the change as is here. I might remove this file later on.

Reply:
Ok!